charts/cluster-autoscaler/Chart.yaml

```diff
@@ -11,4 +11,4 @@ name: cluster-autoscaler
 sources:
   - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
 type: application
-version: 9.43.1
+version: 9.43.2
```

Comment:
You'll need to bump the version again anyway as we've just merged another PR, but can you bump the minor version as we're adding new functionality with this please?

Reply:
@gjtempleton Here we go